src/components/signup/sagas.js

```diff
@@ -25,7 +25,7 @@ function* signupUser(action) {
         if(response.ok){
             const data = yield response.json();
             yield effects.put(signupSuccess());
-            yield effects.put(setCookie(data));
+            yield effects.put(getUserData(data));
```

Comment:
Are you getting the user data or are you setting it on the redux store?

Reply:
Am setting it on the redux store. :)